drakrun/drakrun/lib/postprocessing/capa_plugin/capa_processor.py

```diff
@@ -0,0 +1,373 @@
+import functools
+import itertools
+import logging
+import multiprocessing
+import shutil
+import zipfile
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Any, Dict, Iterator, List, Optional, TextIO, Tuple, Union
+
+import capa.engine as ce
+import capa.features.address as ca
+import capa.render.result_document as crd
+import orjson
+from capa.capabilities.common import find_capabilities
+from capa.features.common import Result
+from capa.features.extractors.base_extractor import ProcessFilter
+from capa.features.extractors.drakvuf.extractor import DrakvufExtractor
+from capa.helpers import get_auto_format
+from capa.loader import get_extractor
+from capa.main import (
+    BACKEND_DOTNET,
+    BACKEND_VIV,
+    FORMAT_DOTNET,
+    OS_WINDOWS,
+    UnsupportedFormatError,
+)
+from capa.rules import Rule, RuleSet, get_rules, get_rules_and_dependencies
+
+# rules related configuration
+capa_rules_dir = Path("./capa-rules")
+
+# analysis configuration
+analyze_malware_pids_only = False
+perform_static_analysis = False
+perform_dynamic_analysis = True
```

Comment:
Is it really something we want to be configurable? If yes, it should be a part of configuration. If no, we should avoid if's that are always False or True (I doubt that someone will modify these constants)